ravenpy/models/raven-hypr/raven-hypr.rvc

```diff
@@ -0,0 +1,13 @@
+#########################################################################
+:FileType          rvc ASCII Raven 3.0.4
+:WrittenBy         Mahkameh Taheri, Juliane Mai & James Craig
+:CreationDate      Feb 2021
+#
+# Emulation of the HYPR model for Salmon River near Prince George
+#------------------------------------------------------------------------
+
+#  Raven Initial Conditions file
+
+:UniformInitialConditions SOIL[0] 10
+:UniformInitialConditions SOIL[1] 10
+:UniformInitialConditions DEPRESSION 20
```

Comment:
We'll probably want to eventually template the initial conditions to be able to control them from RavenPy, ie using  
```
{hru_states_cmd}

{basin_states_cmd}
```
But we can do that later.

Reply:
Probably. We just need to make sure that we enable that these can be calibratable parameters. On top of being calibratable, they are usually derived parameters (like half soil depth in mm) in the calibration setup. But I am sure we can make this work. 

I just tried to use something similar in the CANADIANSHIELD model with the `area`. It us within a `hru_cmd` and it is a derived parameter... So, it's possible; just too complex for me to find the correct syntax and way to use it best.